module-2/backend/week_one.md

```diff
@@ -5,30 +5,112 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+
+  post, get, put, delete, options
+
 2. What is Sinatra?
+
+  Sinatra is a MVC based web framework
+
 4. What is MVC?
-5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+
+  Models, Views, Controllers
+
+5. Why do we follow conventions when creating our actions/path names in our Sinatra routes? 
+
+  they accomplish a simplified way to read the path names and make connections between desired actions
+
 6. What types of variables are accessible in our view templates without explicitly passing them?
+
+  the variables can represent the methods in the models, routes 

```

Comment:
We wouldn't normally store a method in a variable, but we could save an instance of the model itself, which would then give us access to its methods.

Re-read the question, though. This is asking what **type** of variables we can access in our views. The answer to that is **instance variables** (which you show in your answer to number 7).